pkg/resources/galley/configmap.go

```diff
@@ -38,28 +38,28 @@ func (r *Reconciler) configMap() runtime.Object {
 	}
 
 	if util.PointerToBool(r.Config.Spec.Galley.ConfigValidation) {
-		configmap.Data["validatingwebhookconfiguration.yaml"] = r.validatingWebhookConfig(r.Config.Namespace)
+		configmap.Data["validatingwebhookconfiguration.yaml"] = r.validatingWebhookConfig()
 	}
 
 	return configmap
 }
 
-func (r *Reconciler) validatingWebhookConfig(ns string) string {
+func (r *Reconciler) validatingWebhook() runtime.Object {
 	fail := admissionv1beta1.Fail
 	se := admissionv1beta1.SideEffectClassNone
-	webhook := admissionv1beta1.ValidatingWebhookConfiguration{
+
+	return &admissionv1beta1.ValidatingWebhookConfiguration{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      webhookName,
-			Namespace: ns,
```

Comment:
I guess the namespace is omitted from here deliberately, but just to double check, is it? :)

Reply:
Yes, the admission webhooks are cluster scoped resources.

Reply:
Thanks!